src/components/SelectCustomerInstitutionDialog.tsx

```diff
@@ -80,6 +95,18 @@ export const SelectCustomerInstitutionDialog = ({ allowedCustomerIds }: SelectCu
           loading={isLoadingCustomers}
           onChange={(_, value) => setSelectedCustomer(value)}
           isOptionEqualToValue={(option, value) => option.id === value.id}
+          renderOption={(props, option) => {
+            const organization = organizations && organizations.find((org) => org.id === option.cristinId);
+            return organization ? (
+              <OrganizationRenderOption key={organization.id} props={props} option={organization} />
+            ) : (
+              <li {...props}>
+                <Typography fontWeight="bold" key={option.id}>
+                  {option.displayName}
+                </Typography>
+              </li>
```

Comment:
kommer man egentlig noen gang hit? Når i så fall?

Reply:
la den inn for i tilfelle orgQuery skulle feile. Usikker på om det da defaulter tilbake til `getOptionsLabel` 🤔